content/docs/user-guide/experiment-management/persisting-experiments.md

```diff
@@ -0,0 +1,93 @@
+# Persisting Experiments
+
+DVC runs experiments outside of the Git stage/commit cycle for quick iteration.
```

Comment:
This motivation is not very clear. Let's just use the text from https://dvc.org/doc/user-guide/experiment-management#persistent-experiments probably

> When your experiments are good enough to save or share, you may want to store them persistently as Git commits in your repository.